.github/workflows/release_asset.yml

```diff
@@ -35,6 +35,23 @@ jobs:
         with:
           platform: x64
 
+      #fix https://github.com/rooch-network/rooch/issues/1157  with https://github.com/diesel-rs/diesel/blob/master/.github/workflows/ci.yml
+      - name: Install sqlite (Windows)
+        if: matrix.platform == 'windows-latest'
+        shell: cmd
+        run: |
+          choco install sqlite
+          cd /D C:\ProgramData\chocolatey\lib\SQLite\tools
+          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
```

Comment:
Does this require specifying the windows version?

Reply:
Yes, use windows-lastest should be ok.

Reply:
If Windows is upgraded, Microsoft Visual Studio\2022 will also be upgraded. For example, if it reaches Microsoft Visual Studio\2023, an error will be reported.

Reply:
> If Windows is upgraded, Microsoft Visual Studio\2022 will also be upgraded. For example, if it reaches Microsoft Visual Studio\2023, an error will be reported.

That makes sense, I'll try modifying it and see.